docs/dictionary-reference.md

```diff
@@ -67,27 +67,27 @@ In addition to schemas, a Lectern Dictionary can contain reference values that c
+| Property       | Required | Default                | Type                                                | Description                                                                                                                                                                                                                                                                                                                                                              | Example                                              |
+| -------------- | -------- | ---------------------- | --------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `name`         | Required |                        | NameString (no whitespace or `.`)                   | Name of the field. This will be used as the header in TSV files in this field's schema, and in any paths referencing this field.                                                                                                                                                                                                                                         | `"example_field`                                     |
+| `valueType`    | Required |                        | [Field Data Type](#field-data-types)                | Type of value stored in this field                                                                                                                                                                                                                                                                                                                                       | `"string"`                                           |
+| `delimiter`    | Optional | `,`                    | `string`                                            | Character or string that will be used to split multiple values into an array. The default delimiter is a comma `,`. Any characters can be used as a delimiter. The delimiter value can be one or more characters long, but cannot be an empty string. Note: This property has no effect unless the field has `isArray: true`.                                            | `"\|"`                                               |
```

Comment:
Just to double check and confirm, I assume there's validation and TS preventing a submission where `valueType: 'integer', isArray: true, delimiter: '0'` ?
isArray is true but valueType is not string

Reply:
There is not. That is a valid delimiter, and it would prevent numeric values with `0` in them. I would advise against using this delimiter but the spec allows it atm.